packages/components/src/search-control/index.tsx

```diff
@@ -120,7 +120,7 @@ function UnforwardedSearchControl(
 				value={ value ?? '' }
 				prefix={
 					<InputControlPrefixWrapper variant="icon">
-						<Icon icon={ search } fill="currentColor" />
+						<StyledIcon icon={ search } fill="currentColor" />
```

Comment:
<img width="265" alt="SearchControl with flipped search icon" src="https://github.com/user-attachments/assets/a17528cf-4b60-4234-a40a-0d48b68f5792" />

<img width="264" alt="SearchControl in RTL" src="https://github.com/user-attachments/assets/c6fec3f5-3dda-423e-8f6c-7f4f058a2974" />